backend/src/appointment/routes/schedule.py

```diff
@@ -185,6 +185,7 @@ def read_schedule_availabilities(
         owner_name=subscriber.name,
         slots=actual_slots,
         slot_duration=schedule.slot_duration,
+        booking_confirmation=schedule.booking_confirmation
```

Comment:
We should really dedicate a schema to this endpoint at somepoint 😄

Reply:
Yes we should. This is kind of grown from the single-Appointments we used to have